protos/apparmor-prompting.proto

```diff
@@ -22,12 +22,48 @@ message PromptReply {
 }
 
 message PromptReplyResponse {
-    PromptReplyType prompt_reply_type = 1;
-    string message = 2;
-    enum PromptReplyType {
-        SUCCESS = 0;
-        UNKNOWN = 1;
-        PROMPT_NOT_FOUND = 2;
+    string message = 1;
+    oneof data {
```

Comment:
After dealing with the generated code, re-using the `PromptReplyType` name might have been a better choice. Not sure if it's worth changing again now, though